bin/codemods/src/single-tree-rendering.js

```diff
@@ -81,15 +82,37 @@ export default function transformer( file, api ) {
 		} )
 		.remove();
 
-	// Remove empty `import 'lib/react-helpers'`
-	root
+	// Find empty `import 'lib/react-helpers'`
+	let orphanImportHelpers = root
 		.find( j.ImportDeclaration, {
 			source: {
 				value: 'lib/react-helpers',
 			},
 		} )
-		.filter( p => ! p.value.specifiers.length )
-		.remove();
+		.filter( p => ! p.value.specifiers.length );
+
+	if ( orphanImportHelpers.size() ) {
+		// Find out if import had comment above it
+		let comment = _.get( orphanImportHelpers.nodes(), '[0].comments[0]', false );
+
+		// Remove empty `import 'lib/react-helpers'` (and any comments with it)
+		orphanImportHelpers.remove();
+
+		// Put back that removed comment (if any)
+		if ( comment ) {
+			// Find internal dependencies and place comment above first one
+			root
+				.find( j.ImportDeclaration )
+				.filter( p => {
+					let importValue = _.get( p, 'value.source.value', '' );
+					return importValue.indexOf( '/' ) > -1;
```

Comment:
That check might not be entirely accurate -- I think we have (very few) instances where we import something from an external dependencies subdir. (Those cases might be rare enough to not collide with cases where `lib/react-helpers` is the first import below the `Internal Dependencies` block.)

Reply:
Ah, interesting! I didn't think about this.

Suppose this would be only from `./node_modules`? I can see [that happening here and there](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=node_modules&type=) but never within `./client` directory.

Reply:
Well, one example that comes to mind is `react-dom/server` (only inside `server/`) and `react-dom/test-utils` (only for tests). I think there were some more tho...

Reply:
Found one more: `react-pure-render/shallowEqual`. Arguably deprecated tho.

Reply:
Aight, I think [this check is](https://github.com/Automattic/wp-calypso/pull/19518/files#diff-510d4ca2c9f67544a60559465c2d11efR134) good & ready now for another look.

I found a number of internal deps without `/` (namely `config`, `controller`, `dispatcher`, `layout` and `sections`) and then whitelisted these.

Then I started blacklisting external ones you found, until I figured I'll just pull deps from `package.json`. Finally I realised this must not be first time someone has this issue with codemods. Alas, I found a nice solution from `sort-imports.js` where we are checking against external deps.

I'm not sure if any of those files with odd external imports would really be our concern, but what ever kind of imports there are in those files — this change will keep them in bay. One thing less to worry about. :-)